docs/Adding-Software-to-Homebrew.md

```diff
@@ -14,7 +14,7 @@ If everything checks out, you're ready to get started on a new formula!
 
 ### Writing the formula
 
-1. It's a good idea to find existing formulae in Homebrew that have similarities to the software you want to add. This will help you to understand how specific languages, build methods, etc. are typically handled.
+1. It's a good idea to find existing formulae in Homebrew that have similarities to the software you want to add. This will help you to understand how specific languages, build methods, etc. are typically handled. Start by downloading a local clone of the `homebrew/core` tap by running `brew tap --force homebrew/core`.
```

Comment:
```suggestion
1. It's a good idea to find existing formulae in Homebrew that have similarities to the software you want to add. This will help you to understand how specific languages, build methods, etc. are typically handled. Start by cloning the `homebrew/core` tap by running `brew tap --force homebrew/core`.
```